server/migrations/20211207195840-deprecate-offchain.js

```diff
@@ -0,0 +1,403 @@
+'use strict';
+
+const offChainCommunities = [
+  'clcg-commonwealth',
+  'dtrade',
+  'decent',
+  'webb',
+  'agency',
+  'kabocha',
+  'cmn-protocol',
+  'polkaswapcommunity',
+  'stargaze',
+  'cabindao',
+  'ion',
+  'nippon-powder',
+  'endaoment',
+  'poof-cash',
+  'pontem-network',
+  'ubeswap',
+  'euler-finance',
+  'be-your-own-bank',
+  'antimatter-dao',
+  'longhorndao',
+  'magicklu',
+  'burnt-finance',
+  'element-finance',
+  'druside',
+  'sentinel',
+  'gopixel',
+  'anmol',
+  'harvest',
+  'impactmarket',
+  'immunefi',
+  'infinity',
+  'arvex28',
+  '222',
+  'digm-dao',
+  'usemate',
+  'qnt-dao',
+  'knoxedge',
+  'likecoin',
+  'pfeilstorch',
+  'redacted-cartel'
+];
+
+const offchainToRemove = [
+  'agoric-community',
+  'bauhaus-cw',
+  'blockchainyonsei',
+  'bv-test',
+  'cow-moon',
+  'cw-kong',
+  'd',
+  'df344sdfg',
+  'diamond-age-dao-sample',
+  'dydx-internal',
+  'gg',
+  'governance-is-near',
+  'hello-world',
+  'hello-world-wow-martians-are-here',
+  'hen',
+  'honest-farmer',
+  'impossible-finance',
+  'internal',
+  'meta',
+  'my-community',
+  'name',
+  'nns',
+  'o-test',
+  'panacea',
+  'paraswap',
+  'soldev',
+  'svgsvgsvgononloadconfirm1',
+  'test',
+  'test222',
+  'testify',
+  'trtr',
+  'w3f-cw',
+  'winky',
+  'yeeeew',
+  'yzs-stash',
+]
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    /**
+     * Copy offchain community to Chains, with appropriate fields filled-in
+     */
+    const fromOffchainToChain = async (t, community) => {
+      // Get rest of the info
+      const chain = await queryInterface.sequelize.query(
+        `SELECT id, name, icon_url, description FROM "OffchainCommunities" WHERE id='${community}'`,
+        { transaction: t }
+      );
+
+      // Make sure it exists
+      if (chain[0].length === 0) {
+        console.log("community not found", community)
+        return;
+      }
+
+      // Parse out information
+      const info = chain[0][0];
+      const id = info['id']
+      const name = info['name']
+      const icon_url = info['icon_url']
+      const description = info['description'].substring(0, 200)
+      const { website, discord, telegram } = info;
```

Comment:
Gonna need to parse out more info: missing `element`, plug a bunch of misc configuration.